_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,336 @@
+---
+id: efbfc7
+name: Auto-updating text content can be paused or stopped
+rule_type: atomic
+
+description: |
+  This rule checks that there are mechanisms to pause or stop the auto-updating of text content
+
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+
+input_aspects:
+  - DOM Tree
+
+authors:
+  - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [text node][] that is [visible][] and a [child][] in the [flat tree][] of an HTML element, whose content changes through an action that was not initiated by the user, anytime after the [readiness][document readiness] of the document the [text node][] belongs to is equal to "complete".
+
+## Expectation 1
+
+For each test target a [mechanism][] is provided to pause or stop the updating of the content of the [text node][].
+
+**Note**: If there is more than one [text node][] with auto-updating content, a single [mechanism][] may be used to pause or stop updating all [text nodes][text node].
+
+## Expectation 2
+
+The [mechanism][] to pause or stop the auto-updating of the content of the [text node][] is [visible][], has an [accessible name][] that is not empty ("") nor only [ASCII whitespace][] , and is [included in the accessibility tree][].
+
+## Assumptions
+
+This rule assumes that the auto-updating of the content is not [essential][], which is listed as valid exception to SC 2.2.2.
```

Comment:
See https://github.com/act-rules/act-rules.github.io/pull/960/files#r332913127.

Reply:
I've updated the assumption to make it clear that when the auto-updating is essential then the rule may produce incorrect results.